.github/workflows/ci.yml

```diff
@@ -22,11 +22,12 @@ jobs:
         source .envrc
         git submodule update --init --recursive
 
-        py.test
+        pytest
 
         # We want to ensure that all notebooks also follow coding standards.
         cd notebooks; flake8-nb; [ $? -eq 0 ] || exit 1; cd ..
         black-nb .; [ $? -eq 0 ] || exit 1
```

Comment:
Please change to 

         black-nb --check .; [ $? -eq 0 ] || exit 1 

.. that was my fault in the intial setup.

Reply:
Throws a (respy-related) error if I go with the change.